Box.V2.Test.Integration/Configuration/Commands/ICommand.cs

```diff
@@ -43,4 +43,17 @@ public interface ICleanupCommand : ICommand
         /// <returns></returns>
         Task Execute(IBoxClient client);
     }
+
+    /// <summary>
+    /// Interface used to create and of resources. It does not perform dispose action. Usable in cases like new file version upload.
```

Comment:
I think the `remove/dispose` phrase is missing after `and`

Reply:
Godd catch!. I improved wording a bit.